content/attacks/posts/2023-04-13-Yearn-Finance.md

```diff
@@ -1,6 +1,8 @@
 ---
 date: 2023-04-13
-attacks/posts/categories: Protocol exploit
+type of attack: "Protocol exploit"
+target entity: "Yearn Finance"
+type of entity: "Blockchain"
```

Comment:
Rewrite as mentioned before, and merge main

Reply:
Type of attack is `Smart Contract Exploit` and type of entities are 'DeFi, Yield Aggregator'
Please pay attention to new headers, especially `type of attack` and `type of entity`